pkg/cmd/roachtest/tests/versionupgrade.go

```diff
@@ -108,7 +108,18 @@ func runVersionUpgrade(ctx context.Context, t test.Test, c cluster.Cluster) {
 		var cancel context.CancelFunc
 		testCtx, cancel = context.WithTimeout(ctx, localTimeout)
 		defer cancel()
-		opts = append(opts, mixedversion.NumUpgrades(1))
+		opts = append(
+			opts,
+			mixedversion.NumUpgrades(1),
+			// Disable separate-proces deployments in local runs, as it is
```

Comment:
Do we have an existing GH issue to reference here?

Reply:
We don't. I expect this test to fail in nightly runs shortly after it's merged. When it does, I want to ask DR to help debug the issue, given we'll have all the artifacts at that point.

Depending on how that goes we can then create a new issue, or fix something in DR land, or just skip separate-process entirely for this test.